packages/front-end/components/Experiment/ChanceToWinColumn.tsx

```diff
@@ -101,7 +101,7 @@ export default function ChanceToWinColumn({
         <>
           {percentFormatter.format(chanceToWin)}
           {sigText !== "" && (
-            <Tooltip text={sigText} className="d-block" tipPosition={"top"}>
+            <Tooltip body={sigText} className="d-block" tipPosition={"top"}>
```

Comment:
This no longer works since the trigger for the tooltip will be a single space, which is hard to hover over.  Instead, we can wrap the entire cell contents in the tooltip to give it a bigger trigger area.